### ISSUE TITLE
Arrival rate drop issue for specific numbers [TypeError: size must be a number >= 0]

### ISSUE THREAD

**Issue by DavidBonsant** [shoreditch-ops/artillery#243]
Trying to simulate peaks with highs and lows gave us an error, we tested 15 to 5 for 2 minutes and 10 seconds, any other number than 15 to 5 seemed to work, a 100 second duration works too.
here is a sample code that causes the error
```
phases:
  -
    duration: 10
    arrivalRate: 5
    rampTo: 15
  -
    duration: 10
    arrivalRate: 15
  - 
    duration: 10
    arrivalRate: 5

```

**Comment by hassy** [comment]
thanks for the report @DavidBonsant

**Comment by laurwang** [comment]
Hi, @hassy.  I'm getting the same problem.  The error is in line 102 of the current artillery-core/lib/phases.js: let probabilities = crypto.randomBytes(spec.duration * 1000 / tick * 1.25);

The size passed to randomBytes needs to be integer, so I think this could be sorted by changing to 
let probabilities = crypto.randomBytes(Math.round(spec.duration * 1000 / tick * 1.25));//or Math.floor.

In the meantime, with the current code, you just need to make sure that the configuration adheres to the condition that duration*ramp*5 is divisible by 4 (and then you get an integer argument, because spec.duration * 1000 / tick * 1.25 = spec.duration * spec.rampTo * 5 / 4).

**Comment by hassy** [comment]
@laurwang thanks, I didn't see this in my GH notifications for some reason.

**Comment by hassy** [comment]
Fixed in Artillery `v1.5.2`, thanks @laurwang

**Comment by laurwang** [comment]
Thanks, @hassy!
